### DISCUSSION TITLE
Formatting of output data file from OpenNFT

### DISCUSSION THREAD

**Discussion by manish-alluri** [OpenNFT/OpenNFT#158] [Q&A] [answered]
Hi all,

We are moving forward with a pilot study and will be using OpenNFT along with Unity which tries to read the data provided by OpenNFT and display it on screen to the participants in real-time. We wanted to know and see if the data generated by opennft can be formatted in a way which could be easily read by Unity using C#. Since C# has specific package libraries to read the data from an file, we wanted to link the data which is generated directly from opennft folder in a way it is easy to display and visualize the values. 

Is there a way this could be done by changing anything in Opennft that would make this possible?

**Comment by tiborauer** [comment, marked as answer]
I suggest reading the data (i.e., the feedback signal) via the UDP socket rather than from the file. It is more language-independent, I suppose.

**Comment by ykoush** [comment]
Right. Also, depends on what exactly do you plan to send to Unity. You could program any data transfer, as data circulate between Matlab and Python on many levels. 
Btw, let me know if you prefer entirely Python version of OpenNFT, it is in private directory so far, but we could share.
